cypress/integration/account_settings/theme_color/custom_colors_sidebarStyles_spec.js

```diff
@@ -0,0 +1,441 @@
+
+        // 7. Exit User Settings
+        cy.get('#accountSettingsHeader > .close').click();
+
+        // 8. Change user-1 status to Do Not Disturb
+        cy.userStatus(2);
+
+        // * Check Do Not Disturb Indicator color
+        cy.get('.dnd--icon').should('have.css', 'fill', 'rgb(129, 65, 65)');
+
+        // 9. Revert user-1 status to Online
+        cy.userStatus(0);
+    });
+
+    it('should change Mention Jewel BG color  and verify color change', () => {
+        // 1. Selecting Sidebar Header Dropdown, Account Settigns, and Display Settings
+        cy.get('#sidebarHeaderDropdownButton').click();
+        cy.get('#accountSettings').click();
+        cy.get('#displayButton').click();
+
+        // 2. Selecting Theme Edit, Custom Theme, and Sidebar Styles dropdown
+        cy.customColors(0, 'Sidebar Styles');
+
+        // 3. Click Mention Jewel BG
+        cy.get('.input-group-addon').eq(11).click();
+
+        // 4. Click on color window to change color
+        cy.get('.saturation-black').click();
+
+        // 5. Click outside of color modal to remove it from view
+        cy.get('#displaySettingsTitle').click();
+
+        // * Check Mention Jewel BG icon color change
+        cy.get('.color-icon').eq(11).should('have.css', 'background-color', 'rgb(129, 65, 65)');
+
+        // 6. Save Mention Jewel BG color change
+        cy.get('#saveSetting').click();
+
+        // 7. Exit User Settings
+        cy.get('#accountSettingsHeader > .close').click();
+
+        // * Check Mention Jewel BG color
+        cy.get('#unreadIndicatorBottom').should('have.css', 'background-color', 'rgb(129, 65, 65)');
+    });
+
+    it('should change Mention Jewel Text color  and verify color change', () => {
+        // 1. Selecting Sidebar Header Dropdown, Account Settigns, and Display Settings
+        cy.get('#sidebarHeaderDropdownButton').click();
+        cy.get('#accountSettings').click();
+        cy.get('#displayButton').click();
+
+        // 2. Selecting Theme Edit, Custom Theme, and Sidebar Styles dropdown
+        cy.customColors(0, 'Sidebar Styles');
+
+        // 3. Click Mention Jewel Text
+        cy.get('.input-group-addon').eq(12).click();
+
+        // 4. Click on color window to change color
+        cy.get('.saturation-black').click();
+
+        // 5. Click outside of color modal to remove it from view
+        cy.get('#displaySettingsTitle').click();
+
+        // * Check Mention Jewel Text icon color change
+        cy.get('.color-icon').eq(12).should('have.css', 'background-color', 'rgb(65, 92, 129)');
+
+        // 6. Save Mention Jewel Text color change
+        cy.get('#saveSetting').click();
+
+        // 7. Exit User Settings
+        cy.get('#accountSettingsHeader > .close').click();
+
+        // * Check Mention Jewel Text color
+        cy.get('#unreadIndicatorBottom').should('have.css', 'color', 'rgb(65, 92, 129)');
+    });
+
+    it('should revert any color changes made', () => {
```

Comment:
This can be removed once you've implemented the `after` hook.

Reply:
removed last test and added `after` hook